packages/scandipwa/src/component/ProductCompareItem/ProductCompareItem.component.js

```diff
@@ -127,35 +128,52 @@ export class ProductCompareItem extends PureComponent {
         );
     }
 
-    renderAddToCartBtnDisabled() {
+    renderAddToCartBtnDisabled(renderLink) {
         const { linkTo, overriddenAddToCartBtnHandler } = this.props;
 
-        return (
-            <Link
-              to={ linkTo }
-              onClick={ overriddenAddToCartBtnHandler }
-              block="ProductCompareItem"
-              elem="AddToCartBtnWrapper"
-            >
-                <AddToCart
-                  product={ {} }
-                  groupedProductQuantity={ {} }
-                  productOptionsData={ {} }
-                  disableHandler
-                  mix={ { block: 'ProductCompareItem', elem: 'AddToCartBtn' } }
-                />
-            </Link>
+        const btn = (
```

Comment:
Wondering if it won't if be better if we actually move this rendering of button into separate function, so when devs will override button they won't need to override also lines 143 - 157?

Reply:
Done.

Reply:
Great. Thanks!